include/battle.h

```diff
@@ -357,19 +357,28 @@ struct BattleResults
 
 extern struct BattleResults gBattleResults;
 
+struct MultiPartnerEnigmaBerry
+{
+    u8 versionSignatureLo;
+    u8 versionSignatureHi;
+    u8 vsScreenHealthFlagsLo;
+    u8 vsScreenHealthFlagsHi;
+    struct BattleEnigmaBerry battleEnigmaBerry;
+};
```

Comment:
Also same as in https://github.com/pret/pokeemerald/pull/1376, why is this `MultiPartnerEnigmaBerry` if only 1 field is related to Enigma Berry